### PR TITLE
Fix WavHandler

### PR DESCRIPTION
**Pull request by AaronWantoch**

### CODE REVIEW

File: test/wavTest.cpp [hunk 1]
```diff
@@ -8,11 +8,11 @@
 
 TEST(wavTest, temp){
     size_t probeSize = 1024;
-    WavHandler wav_hdr("../rsc/test3.wav");
+    WavHandler wav_hdr("../rsc/a_p.wav");
     Segment seg(probeSize);
     std::ifstream correctData;
     float data;
-    float allowedError = 0.1;
+    float allowedError = 10000;
```
Comment (bstrzelecki) [inline test/wavTest.cpp]: What is the reason for such increase in allowedError?

Comment (AaronWantoch) [reply test/wavTest.cpp]: Earlier error was relative now it is absolute

Comment (bstrzelecki) [reply test/wavTest.cpp]: What is the unit then?

Comment (AaronWantoch) [reply test/wavTest.cpp]: I don't know, but in file the smalest value is like 2*10^7 so 10^4 is pretty small compered to that

Comment (bstrzelecki) [reply test/wavTest.cpp]: In this case we should return to relative error

Comment (bstrzelecki) [reply test/wavTest.cpp]: Also in such instance it should be possible to assert to the exact value

Comment (AaronWantoch) [reply test/wavTest.cpp]: > Also in such instance it should be possible to assert to the exact value

I think due to the size the issue might be caused by constraints of float cause from what I remember float is inacurate for big numbers

Comment (AaronWantoch) [reply test/wavTest.cpp]: Changed back to relative error this time of 0.001%

Comment (bstrzelecki) [reply test/wavTest.cpp]: Well yes, but in this solution there is no performed operation that would introduce such inaccuracies

Comment (AaronWantoch) [reply test/wavTest.cpp]: maybe floats saved to file are saved in wrong format

Comment (AaronWantoch) [reply test/wavTest.cpp]: yeah that was that now it asserts exact value

**Review by bstrzelecki — APPROVED** [review]

**Review by BognaLew — APPROVED** [review]

**Pull request merged**